### PR TITLE
Add indexes for hbmember, hbclaim and hbhousehold to DMS

### PR DESCRIPTION
**Pull request by elcorbs**
Adding indexes to tables to hopefully speed up querying against postgres.

### CODE REVIEW

File: terraform/production/selection_rules.json [hunk 1]
```diff
@@ -76,6 +76,63 @@
...
+          "schema-name": "%",
+          "table-name": "hbmember"
+        },
+        "primary-key-def": {
+          "name": "member-unique-idx",
+          "origin": "unique-index",
+          "columns": [
+            "claim_id",
+            "member_id",
+            "house_id"
```
Comment (makeyw) [inline terraform/production/selection_rules.json:96]: ah very nice :)

**Review by makeyw — APPROVED** [review]
Great stuff

**Comment by makeyw** [comment]
Actually, assuming this improves matters we'll likely want to look at the other databases too and ensure they have indexes - i'll add a card.

**Comment by makeyw** [comment]
Also wondering whether we can have indexes on the fields we search on too? (fn/sn etc....).
Might not even be an issue but something to ponder.

**Comment by elcorbs** [comment]
@LBHMKeyworth  Yeah definitely agree with both of those points 👍

**Pull request merged**